test/evergreen.yml

```diff
@@ -68,6 +68,11 @@ functions:
               export UBSAN_OPTIONS="$COMMON_SAN_OPTIONS:print_stacktrace=1"
             fi
 
+            if [ -n "${num_threads}" ]; then
+              num_threads="${num_threads}"
```

Comment:
Its not really the number of threads, but number parallel or number of processes.

Reply:
Done, i renamed it